CONTRIBUTING.md

```diff
@@ -145,32 +261,490 @@ for review, which allows you to sidestep this issue.
 This is not a bulletproof method though, as OfBorg still does review requests even on draft PRs.
 ```
 
-## Backporting changes
+## How to backport pull requests
```

Comment:
This section and subsections were partially rewritten from originally duplicate information.